src/HOCs/shouldComponentUpdateHOC.jsx

```diff
@@ -12,28 +12,26 @@ const ShouldComponentUpdateHOC = (WrappedComponent) => {
       );
 
       const customizer = (oldProp, newProp, key) => {
```

Comment:
Yeah, I've had similar thoughts. I kept it in because it was there before, but it's been suspect. One thing I thought about doing (maybe/not in this PR) was to implement `shouldComponentUpdate` for each component. Initially this thought came from wanting to remove the indirection added by using an HOC for this. Thanks for brining this up, this is a really good point, I can double check that this is essentially no-op/default-op behavior and we can take it out.

Reply:
SGTM